Rakefile

```diff
@@ -1,7 +1,63 @@
+require 'pg'
+
 if ENV['RACK_ENV'] != 'production'
   require 'rspec/core/rake_task'
   
   RSpec::Core::RakeTask.new :spec
   
   task default: [:spec]
 end
+
+task :setup do
```

Comment:
This all looks great.  One way to improve it slightly would be to separate database creation from table creation.  Then you could have tasks like `create_db`, `drop_db`, `create_tables` and `drop_tables` which give more precise control. For example, you could drop the tables without having to drop and recreate the whole db.  You could also have some safeguards on `drop_db` so that you never accidentally drop a db which has tables.

Reply:
Great advice, Eddie, thanks!